packages/client/src/rest/index.ts

```diff
@@ -27,57 +27,8 @@ import {
 import fetch from 'node-fetch';
 import promiseTools from 'promise-tools';
 import { loggerFor } from '../logger';
-const L = loggerFor(__filename);
-
-export interface SchemaServiceRestClient {
-	fetchSchema(schemaSvcUrl: string, name: string, version: string): Promise<Dictionary>;
-	fetchDiff(schemaSvcUrl: string, name: string, fromVersion: string, toVersion: string): Promise<DictionaryDiff>;
-}
-
-export const restClient: SchemaServiceRestClient = {
-	fetchSchema: async (schemaSvcUrl: string, name: string, version: string): Promise<Dictionary> => {
-		// for testing where we need to work against stub schema
-		if (schemaSvcUrl.startsWith('file://')) {
-			return await loadSchemaFromFile(version, schemaSvcUrl, name);
-		}
 
-		if (!schemaSvcUrl) {
-			throw new Error('please configure a valid url to get schema from');
-		}
-		const url = `${schemaSvcUrl}/dictionaries?name=${name}&version=${version}`;
-		try {
-			L.debug(`in fetch live schema ${version}`);
-			const schemaDictionary = await doRequest(url);
-			// todo validate response and map it to a schema
-			return schemaDictionary[0] as Dictionary;
-		} catch (error: unknown) {
-			L.error(`failed to fetch schema at url: ${url} - ${unknownToString(error)}`);
-			throw error;
-		}
-	},
-	fetchDiff: async (
-		schemaSvcBaseUrl: string,
-		name: string,
-		fromVersion: string,
-		toVersion: string,
-	): Promise<DictionaryDiff> => {
-		// TODO: Error handling (return result?)
-		const url = `${schemaSvcBaseUrl}/diff?name=${name}&left=${fromVersion}&right=${toVersion}`;
-		const diffResponse = await doRequest(url);
-
-		const diffArray = DictionaryDiffArray.parse(diffResponse);
-
-		const result: DictionaryDiff = new Map();
-		for (const entry of diffArray) {
-			const fieldName = entry[0];
-			if (entry[1]) {
-				const fieldDiff: FieldDiff = entry[1];
-				result.set(fieldName, fieldDiff);
-			}
-		}
-		return result;
-	},
-};
+const L = loggerFor(__filename);
```

Comment:
is a single character variable name really necessary? is that much better or faster or easier to understand/read than `logger`? of `log`?

Reply:
Inherited code, needs review and likely refactor, not touching in this PR. Only appears added because I removed a layer of nesting in this file.